src/views/dialogs/DepositDialog2/RouteOptions.tsx

```diff
@@ -0,0 +1,123 @@
+import { RouteResponse } from '@skip-go/client';
+import { formatUnits } from 'viem';
+
+import { LightningIcon, ShieldIcon } from '@/icons';
+
+import { DepositSpeed } from './types';
+
+type Props = {
+  routes: { slow?: RouteResponse; fast?: RouteResponse };
+  isLoading: boolean;
+  disabled: boolean;
+  selectedSpeed: DepositSpeed;
+  onSelectSpeed: (route: DepositSpeed) => void;
+};
+
+export const RouteOptions = ({
+  routes,
+  isLoading,
+  selectedSpeed,
+  onSelectSpeed,
+  disabled,
+}: Props) => {
+  // TODO(deposit2.0): finalize error handling here
+  if (!routes.slow && !routes.fast) {
+    return <div>There was a error loading deposit quotes.</div>;
+  }
+
+  const fastOperationFee = // @ts-ignore
+    routes.fast?.operations.find((op) => Boolean(op.goFastTransfer))?.goFastTransfer?.fee;
```

Comment:
Does this only take the first operation with `goFastTransfer`? Is it ok to ignore other ops?

Reply:
oops meant to leave a comment here. they are working up surfacing all the fees in their top level `fees` field in a newer version of the SDK. ill bump when thats ready :D 

but yes in general each swap should have max 1 `fast` operation!